src/components/__tests__/TextArea.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { shallowMount, Wrapper } from "@vue/test-utils";
+import TextArea from "@/components/TextArea.vue";
+
+describe("TextArea.vue", () => {
+  let wrapper: Wrapper<TextArea>;
+  let vm: any;
+
+  beforeEach(() => {
+    wrapper = shallowMount(TextArea, {
+      propsData: {
+        placeholder: "Type text..",
+      },
+    });
+    vm = wrapper.vm;
+  });
+
+  it("does a wrapper exist", () => {
+    expect(wrapper.isVueInstance).toBeTruthy();
```

Comment:
it's ok to play with this test several times, but I wouldn't test it in every component. It doesn't have a lot of value

Reply:
i tried a new for me property isVueInstance instead exists()

Reply:
oh, ok